### PR TITLE
Added Notice Board

### PR DESCRIPTION
**Pull request by BAAHUBALI7781**

### CODE REVIEW

File: src/views/AllNotice.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+}
+.inp{
+    margin:1rem auto;
+    width:90%;
+    max-width:50rem;
+    padding:15px;
+    border-radius:10px;
+    color:black;
+    /* text-align: center; */
+}
```
Comment (shivam-51) [inline src/views/AllNotice.vue]: Why is this CSS required in this file?

File: src/views/AllNotice.vue [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    border: 1px solid #ff0077;
+    background-color: #ff0077;
+    color: white;
+    padding: 0.05rem 1rem;
+    box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.26);
+}
+button:hover{
+    background-color: #ec3169;
+    border-color: #ec3169;
+    box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.26);
```
Comment (shivam-51) [inline src/views/AllNotice.vue:135]: Why??

File: src/views/AllNotice.vue [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      <ul v-if="noticearr.length!==0">
+          <li v-for="not in noticearr" :key="not.id">
+              <div style="text-align:right">
+                  <span style="color:blue"><small>By: {{not.user}}</small></span>
+              </div>
+              <div style="text-align:left">
+                  <span><strong>{{not.notices}}</strong></span>
+              </div>
+              <div style="text-align:right">
+                  <span style="color:blue"><small>{{not.timestamp}}</small></span>
```
Comment (shivam-51) [inline src/views/AllNotice.vue]: Do we want the author name and the time to be at two different locations?
Either make it "Posted by foo on bar" at the top or at the bottom.

**Review by shivam-51 — CHANGES_REQUESTED** [review]

File: src/views/AllNotice.vue (hunk 1 above)
Comment (BAAHUBALI7781) [reply src/views/AllNotice.vue]: I changed input field to text-area but forgot to remove the input css. I will remove that

**Review by shivam-51 — APPROVED** [review]

**Pull request merged**